mne/decoding/transformer.py

```diff
@@ -13,42 +13,113 @@
 from .. import pick_types
 from ..filter import filter_data, _triage_filter_params
 from ..time_frequency.psd import psd_array_multitaper
-from ..externals import six
-from ..utils import _check_type_picks
+from ..externals.six import string_types
+from ..utils import _check_type_picks, check_version
+from ..io.pick import pick_info, _pick_data_channels, _picks_by_type
+from ..cov import _check_scalings_user
 
 
-class Scaler(TransformerMixin):
-    u"""Standardize data across channels.
+class _ConstantScaler():
+    """Scale channel types using constant values."""
+
+    def __init__(self, info, scalings, do_scaling=True):
+        self._scalings = _check_scalings_user(scalings)
+        self._picks_by_type = _picks_by_type(pick_info(
+            info, _pick_data_channels(info, exclude=())))
+        self._do_scaling = do_scaling
+
+    def fit(self, X, y=None):
+        std = np.ones(sum(len(p[1]) for p in self._picks_by_type))
+        if X.shape[1] != len(std):
+            raise ValueError('info had %d data channels but X has %d channels'
+                             % (len(self._std), len(X)))
+        if self._do_scaling:  # this is silly, but necessary for completeness
+            for kind, picks in self._picks_by_type:
+                std[picks] = 1. / self._scalings[kind]
+        self.std_ = std
+        self.mean_ = np.zeros_like(std)
+        return self
+
+    def transform(self, X, y=None):
+        return X / self.std_
+
+    def inverse_transform(self, X, y=None):
+        return X * self.std_
+
+    def fit_transform(self, X, y=None):
+        return self.fit(X, y).transform(X)
+
+
+def _sklearn_reshape_apply(func, return_result, X, *args):
+    """Reshape epochs and apply function."""
```

Comment:
...oh, I think I can explain the difference. `Vectorizer` would treat each channel *and time point* independently since they are all features, just scaling each one across its set of epochs (right?). This will scale each channel, including all time points from all epochs from it together.